dist/index.html

```diff
@@ -5,35 +5,37 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <link
-    href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
-    rel="stylesheet" />
+  href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
+  rel="stylesheet" />
   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
-    integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
-    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+  crossorigin="anonymous" referrerpolicy="no-referrer" />
   <title>Movies App</title>
 <script defer src="main.js"></script><link href="main.css" rel="stylesheet"></head>
 <body>
   <header>
-      <nav class="navbar">
-        <a href="#" class="nav-logo">
-          <i class="fa-solid fa-film"></i> 
-          <p>TV SHOW</p>
-        </a>
-        <ul class="menu-list">
-          <li class="list-item">
-            <a href="#">Home page</a>
-          </li>
-          <li class="list-item">
-            <a href="#">About us</a>
-          </li>
-          <li class="list-item">
-            <a href="#">Downloads</a>
-          </li>
-        </ul>
+    <nav class="navbar">
+      <a href="#" class="nav-logo">
+        <i class="fa-solid fa-film"></i>
+        <p>TV SHOW</p>
+      </a>
+      <ul class="menu-list">
+        <li class="list-item">
+          <a href="#">Home page</a>
+        </li>
+        <li class="list-item">
+          <a href="#">About us</a>
+        </li>
+        <li class="list-item">
+          <a href="#">Downloads</a>
+        </li>
+      </ul>
     </nav>
```

Comment:
[Optional] It would be better if you change these links to some show categories. For example, `Drama`, `Adventure`, and `Thrill`. Also clicking on one of these links will show a new list of shows from the category type specified in the link.

Reply:
Super cool suggestion, I will implement that.